great_expectations/datasource/data_connector/inferred_asset_glue_catalog_data_connector.py

```diff
@@ -0,0 +1,194 @@
+import logging
+from typing import Any, Dict, Iterator, List, Optional, Tuple
+
+from great_expectations.datasource.data_connector.configured_asset_glue_catalog_data_connector import (
+    ConfiguredAssetGlueCatalogDataConnector,
+)
+from great_expectations.exceptions import DataConnectorError
+from great_expectations.execution_engine import ExecutionEngine
+
+logger = logging.getLogger(__name__)
+
+
+class InferredAssetGlueCatalogDataConnector(ConfiguredAssetGlueCatalogDataConnector):
+    """
+    The InferredAssetGlueCatalogDataConnector is one of two classes (ConfiguredAssetGlueCatalogDataConnector being the
+    other one) designed for connecting to data through AWS Glue Data Catalog.
+
+    It connects to assets (database and tables) inferred from AWS Glue Data Catalog.
+
+    InferredAssetGlueCatalogDataConnector that operates on AWS Glue Data Catalog and determines
+    the data_asset_name implicitly (e.g., by listing all database and table names from Glue Data Catalog)
+    """
+
+    def __init__(
```

Comment:
@lccasagrande Could you please look through your implementation of this class (`InferredAssetGlueCatalogDataConnector`) and see which methods could be combined with the methods having the same name in the parent class, without the loss of generality for all SQL type data connectors, and thus reduce the number of method overrides needed as part of implementing the present class.  Thank you.

Reply:
Done with code refactoring